storage/innobase/xtrabackup/doc/source/xtrabackup_bin/restoring_individual_tables.rst

```diff
@@ -86,14 +80,14 @@ Importing the Table
 
 On the destination server running |Percona Server| with |XtraDB| and
 `innodb_import_table_from_xtrabackup <http://www.percona.com/doc/percona-server/5.5/management/innodb_expand_import.html#innodb_import_table_from_xtrabackup>`_
-option enabled, or |MySQL| 5.6, create a table with the same structure, and
+option enabled, or |MySQL| 5.7, create a table with the same structure, and
 then perform the following steps:
```

Comment:
We don't have innodb_import_table_from_xtrabackup
```suggestion
On the destination server, create a table with the same structure, and
then perform the following steps:
```

Reply:
Updated.